### PR TITLE
correction to instructions as to which build script to edit

### PR DESCRIPTION
**Pull request by jamesrwoodruff**

### CODE REVIEW

File: contents/index.adoc [hunk 1]
```diff
@@ -219,7 +219,7 @@ $ mkdir -p greeter/src/main/java/greeter
 include::{groovy-example-dir}/greeter/src/main/java/greeter/Greeter.java[]
 ----
 
-As the target is a Java application, you also need to tell Gradle the name of the class which is the entry point. Edit the root project build script file again and assign the `mainClassName` property to Java class that contains the `main` method.
+As the target is a Java application, you also need to tell Gradle the name of the class which is the entry point. Edit the build script of the greeter sub-project and assign the `mainClassName` property to Java class that contains the `main` method.
```
Comment (das-g) [inline contents/index.adoc]:
```suggestion
As the target is a Java application, you also need to tell Gradle the name of the class which is the entry point. Edit the build script of the `greeter` sub-project and assign the `mainClassName` property to Java class that contains the `main` method.
```

**Comment by lacasseio** [comment]
The PR isn't signed off so I manually did the edit. Thanks for pointing the issue out.

**Comment by das-g** [comment]
> The PR isn't signed off so I manually did the edit. Thanks for pointing the issue out.

For posterity: That's commit https://github.com/gradle-guides/creating-multi-project-builds/commit/66ee76c12175bd2a797be48d7595412c8f1098b6.

Though, I think the original commit messages of 3121cf1 and 89ea687 would have been much more accurate. (This change is at best a clarification change, probably even a semantic correction. It's not about mistyped words.)

**Comment by lacasseio** [comment]
Right, I didn't notice the DCO change failed because of the Co-Authored-by tag. Thanks for pointing out the clarification.

**Pull request closed without merging**